src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersResponseWithoutRoles.java

```diff
@@ -21,12 +26,18 @@ public class ProfessionalUsersResponseWithoutRoles {
     public String email;
     @JsonProperty
     public String idamStatus;
+    @JsonProperty
+    private LocalDateTime lastUpdated;
+    @JsonProperty
+    public List<UserAccessType> userAccessTypes = new ArrayList<>();
```

Comment:
```suggestion
    public Set<UserAccessType> userAccessTypes = new HashSet<>();
```

should this be a Set instead of a List? - noticed you went for Set in GA-24 for `UserAccessType`